readme.md

```diff
@@ -124,17 +124,23 @@ If you don't have root access, just install everything on your user's folder and
 
 This is an example on how to record the system update package count daily.  
 This will update the file `/var/tmp/updatecount.txt` for later access.  
-Root privilege is required for this operation.
+Root privilege is required for this operation. 
+Only /etc/crontab and the files in /etc/cron.d/ have a username field.
+ 
+##### So there you can do this:
 
-##### To edit root's crontab
 ```bash
-sudo crontab -e
+sudo vim /etc/crontab
+# FireMotD system updates check (randomly execute between 0:00:00 and 5:59:59)
+0 0 * * * root perl -e 'sleep int(rand(21600))' && /usr/local/bin/FireMotD -S &>/dev/null
 ```
 
-##### Then add this line
+##### From root's crontab you can use:
```

Comment:
Again my personal opinion: I would prefer the above solution and leave this out completely. I barely see it used and this method is less clean compared to plain files below `/etc`. The command `crontab -e` is a good solution for non-sudo system users, which firemotd will not be useful for.